extensions/core/deployment/src/main/java/org/apache/camel/quarkus/core/deployment/CamelBeanBuildItem.java

```diff
@@ -20,18 +20,18 @@
 
 import io.quarkus.builder.item.MultiBuildItem;
 
-public final class CamelRegistryBuildItem extends MultiBuildItem {
+public final class CamelBeanBuildItem extends MultiBuildItem {
```

Comment:
A short class level javadoc stating the purpose and limitations would be nice. Something like

> A MultiBuildItem holding beans to add to CamelRegistry immediately after the registry is created. Can be produced only by methods that do not depend on CamelRuntime (because otherwise there is a circular dependency).